### ISSUE TITLE
termux-notification - multiline doesn't work

### ISSUE THREAD

**Issue by ourarash** [termux/termux-api#274]
**Problem description**
Termux notifications cuts the line and doesn't show strings that don't fit in a line. Using `\n` or piping to stdin as recommended [here](https://github.com/termux/termux-api/issues/32#issuecomment-271443823) doesn't work either. In fact it looks like the piped value is ignored.

**Steps to reproduce**
```bash
ls $PREFIX | termux-notification --title 'Some folders'
```

**Expected behavior**
Wrap the strings that don't fit in a single line.

**Additional information**
Non-rooted device. Galaxy Note 8.
```
$ termux-info                                                 Updatable packages:
All packages up to date
Subscribed repositories:
https://dl.bintray.com/grimler/science-packages-21 science/stable
https://dl.bintray.com/grimler/game-packages-21 games/stable
https://termux.net stable/main
https://termux.net stable/main
System information:
Linux localhost 4.9.112-15119493 #2 SMP PREEMPT Thu May 30 19:02:31 KST 2019 aarch64 Android
Termux-packages arch:
aarch64
Android version:
9
Device manufacturer:
samsung
Device model:
SM-N960
```

The below image shows the of the above command results on my device.

![image](https://user-images.githubusercontent.com/20846105/61596288-22567600-abb6-11e9-896d-13965b0fba9f.png)
---
![image](https://user-images.githubusercontent.com/20846105/61596224-98a6a880-abb5-11e9-95c7-950ca1444aa5.png)

**Comment by xalexalex** [comment]
Due to a regression, currently stdin doesn't work to set termux-notification's content (#271).

However, multiline still doesn't work:

```
#doesn't work
termux-notification --content "line 1\nline 2"

#also doesn't work
termux-notification --content "line 1
line 2"
```

**Comment by ourarash** [comment]
Any chance this will be fixed?

**Comment by xalexalex** [comment]
Yeah, just submitted a PR that fixes it. However I still have to look into #271 (and termux/termux-api#71) which is a closely related issue

**Comment by fornwall** [comment]
Thanks for reporting! Fixed in version 0.44 of the termux-api package.

**Comment by ourarash** [comment]
Thanks for fixing! How does the multiline work now? Does it need `\n` or will it wrap automatically?

**Comment by xalexalex** [comment]
> Thanks for fixing! How does the multiline work now? Does it need `\n` or will it wrap automatically?
Just tested and:

- long lines don't wrap, they just get truncated by `...`
- `\n` doesn't work. Did it work? (is it a regression?)
- actually embedding a newline in the string works. i.e.
```
termux-notification -c 'hello
world'
```

> Also, when will this version be available on google Play?

`termux-api-package` does not end up in google play; just `pkg up` in termux to update it

**Comment by xeffyr** [comment]
> \n doesn't work. Did it work? (is it a regression?)

Just `\n` will not work unless it is parsed and replaced with newline character.
```
[xeffyr]:~:$ echo "a\nb"
a\nb
[xeffyr]:~:$ echo -e "a\nb"
a
b
[xeffyr]:~:$
```

**Comment by xalexalex** [comment]
Whoops, sorry, I was referring to the argument to `--content` and had forgotten about stdin entirely.

So `termux-notification --content 'a\nb'` won't work, but `echo -e 'a\nb' | termux-notification` will.

**Comment by ourarash** [comment]
> * `\n` doesn't work. Did it work? (is it a regression?)
> * actually embedding a newline in the string works. i.e.

It now works if called from a node script. I used it in my package [stock-crypto-monitor](https://www.npmjs.com/package/termux-api) to show a multi-line notification. Thanks for the fix!

**Comment by langrenfengzi** [comment]
"\n" works, but auto-wrap still don't.

**Comment by xalexalex** [comment]
Confirmed; reopening
